### PR TITLE
fix: Do not throw message for no adjustment associated with special l…

### PR DESCRIPTION
**Pull request by ap891843**
…evels.

Signed-off-by: Aman Prashant <aman.prashant@broadcom.com>

### CODE REVIEW

File: server/dialect-idms/src/main/java/org/eclipse/lsp/cobol/dialects/idms/IdmsDialect.java [hunk 1]
```diff
@@ -206,7 +209,8 @@ private Collection<SyntaxError> getWarningForNoAdjustment(
       Pair<Range, Integer> rangeIntegerPair,
       String uri,
       Locality sourceLocality) {
-    if (shouldLevelNotBeAdjusted(copybookLevel, firstLevel, rangeIntegerPair.getRight())) {
+    if (shouldLevelNotBeAdjusted(copybookLevel, firstLevel, rangeIntegerPair.getRight())
+        && !SPECIAL_LEVELS.contains(rangeIntegerPair.getRight())) {
```
Comment (ishche) [inline server/dialect-idms/src/main/java/org/eclipse/lsp/cobol/dialects/idms/IdmsDialect.java:213]: It's better to move it into the shouldLevelNotBeAdjusted method.

Comment (ap891843) [reply server/dialect-idms/src/main/java/org/eclipse/lsp/cobol/dialects/idms/IdmsDialect.java]: Updated

**Review by Nurkambay — APPROVED** [review]

**Review by ishche — APPROVED** [review]
LGTM. Is it IDMS-specific?

**Comment by ap891843** [comment]
> LGTM. Is it IDMS-specific?

Yes

**Pull request merged**